rm/rm-node/src/main/java/org/ow2/proactive/resourcemanager/nodesource/dataspace/DataSpaceNodeConfigurationAgent.java

```diff
@@ -161,7 +161,7 @@ public boolean configureNode() {
         }
 
         startCacheSpace();
-        PAActiveObject.terminateActiveObject(false);
+//        PAActiveObject.terminateActiveObject(false);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)